### PR TITLE
factory-package-news: support newer python3 rpm bindings

### PR DESCRIPTION
**Pull request by g7**
rpm's python bindings changed in version 4.15 [0] so that they actually return utf-8 strings. Handle this case while keeping support for the older bindings such as the ones shipped with Leap 15.

[0] https://github.com/rpm-software-management/rpm/commit/84920f898315d09a57a3f1067433eaeb7de5e830

### CODE REVIEW

**Comment by g7** [comment]
v2: addressed flake8 warnings :)

File: factory-package-news/factory-package-news.py [hunk 1]
```diff
@@ -18,6 +18,15 @@
 changelog_max_lines = 100  # maximum number of changelog lines per package
 
 
+# rpm's python bindings changed in version 4.15 [0] so that they actually return
+# utf-8 strings. Leap 15 ships an older version, so this is needed to
+# keep this script working there too.
+#
+# [0] https://github.com/rpm-software-management/rpm/commit/84920f898315d09a57a3f1067433eaeb7de5e830
+def forcestr(content):
```
Comment (dirkmueller) [inline factory-package-news/factory-package-news.py:26]: I wonder if `utf8str` would be a slightly besser name.

**Review by dirkmueller — APPROVED** [review]
LGTM, small nit inline

**Comment by dirkmueller** [comment]
test are passing, we can solve the nit anytime..

File: factory-package-news/factory-package-news.py (hunk 1 above)
Comment (g7) [reply factory-package-news/factory-package-news.py:26]: Thanks, yeah definitely looks better than `forcestr` :) Should I make another PR that changes that?

Comment (dirkmueller) [reply factory-package-news/factory-package-news.py:26]: Yes please if you agree

**Pull request merged**